app/main.py

```
@@ -1 +1,16 @@
-# write your code here
+class FileNotFound():
+    """If file not found return"""
```

Issue: The `FileNotFound` class is defined but never used in the code. If the task requires a custom exception for file-not-found cases, you should raise this exception in the `except FileNotFoundError` block and ensure it inherits from `Exception`.